calculate_average_jbachorik.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+source ~/.sdkman/bin/sdkman-init.sh > /dev/null
+sdk use java 21.0.1-tem > /dev/null
```

Comment:
Can you please move this to `prepare_jbachorik.sh` (see baseline for an example; we extracted this to run everything with hyperfine and keep the `sdk` call out of the timed runs).